src/WebJobs.Script.Abstractions/Description/FunctionMetadata.cs

```diff
@@ -7,7 +7,7 @@
 using System.IO;
 using System.Linq;
 
-namespace Microsoft.Azure.WebJobs.Script.Description
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Description
```

Comment:
Before we release the new package with all these new types, I think we should do a final pass on the public surface area. These types were never meant for public consumption, and there are lots of public helpers and other methods that I think we can move from the public surface area. E.g. FunctionMetadata.IsHttpInAndOutFunction, BindingMetadata.Create, etc. With these being elevated as a new extensibility model, we should review and only include what has to be there. We can do this in a separate PR - doesn't need to block this.